---
title: AsposeArgumentOutOfRangeException
second_title: Справочник по Aspose.Email для .NET API
description: Инициализирует новый экземпляр классаAsposeArgumentOutOfRangeExceptionaspose.email/asposeargumentoutofrangeexception.
type: docs
weight: 10
url: /ru/net/aspose.email/asposeargumentoutofrangeexception/asposeargumentoutofrangeexception/
---
## AsposeArgumentOutOfRangeException() {#constructor}

Инициализирует новый экземпляр класса[`AsposeArgumentOutOfRangeException`](../../asposeargumentoutofrangeexception).

```csharp
public AsposeArgumentOutOfRangeException()
```

### Смотрите также

* class [AsposeArgumentOutOfRangeException](../../asposeargumentoutofrangeexception)
* пространство имен [Aspose.Email](../../asposeargumentoutofrangeexception)
* сборка [Aspose.Email](../../../)

---

## AsposeArgumentOutOfRangeException(string) {#constructor_2}

Инициализирует новый экземпляр класса[`AsposeArgumentOutOfRangeException`](../../asposeargumentoutofrangeexception)с указанным сообщением об ошибке.

```csharp
public AsposeArgumentOutOfRangeException(string message)
```

| Параметр | Тип | Описание |
| --- | --- | --- |
| message | String | Сообщение, описывающее ошибку. |

### Смотрите также

* class [AsposeArgumentOutOfRangeException](../../asposeargumentoutofrangeexception)
* пространство имен [Aspose.Email](../../asposeargumentoutofrangeexception)
* сборка [Aspose.Email](../../../)

---

## AsposeArgumentOutOfRangeException(string, params object[]) {#constructor_5}

Инициализирует новый экземпляр класса[`AsposeArgumentOutOfRangeException`](../../asposeargumentoutofrangeexception)с указанным сообщением об ошибке.

```csharp
public AsposeArgumentOutOfRangeException(string format, params object[] parameters)
```

| Параметр | Тип | Описание |
| --- | --- | --- |
| format | String | Формат сообщения |
| parameters | Object[] | Параметры формата |

### Смотрите также

* class [AsposeArgumentOutOfRangeException](../../asposeargumentoutofrangeexception)
* пространство имен [Aspose.Email](../../asposeargumentoutofrangeexception)
* сборка [Aspose.Email](../../../)

---

## AsposeArgumentOutOfRangeException(string, int) {#constructor_3}

```csharp
public AsposeArgumentOutOfRangeException(string format, int parameter)
```

### Смотрите также

* class [AsposeArgumentOutOfRangeException](../../asposeargumentoutofrangeexception)
* пространство имен [Aspose.Email](../../asposeargumentoutofrangeexception)
* сборка [Aspose.Email](../../../)

---

## AsposeArgumentOutOfRangeException(string, Exception) {#constructor_4}

Инициализирует новый экземпляр класса[`AsposeArgumentOutOfRangeException`](../../asposeargumentoutofrangeexception) с указанным сообщением об ошибке и ссылка на внутреннее исключение, которое является причиной этого исключения.

```csharp
public AsposeArgumentOutOfRangeException(string message, Exception innerException)
```

| Параметр | Тип | Описание |
| --- | --- | --- |
| message | String | Сообщение об ошибке, объясняющее причину исключения. |
| innerException | Exception | Исключение, являющееся причиной текущего исключения, или пустая ссылка (Nothing в Visual Basic), если нет внутреннего исключения указано. |

### Смотрите также

* class [AsposeArgumentOutOfRangeException](../../asposeargumentoutofrangeexception)
* пространство имен [Aspose.Email](../../asposeargumentoutofrangeexception)
* сборка [Aspose.Email](../../../)

---

## AsposeArgumentOutOfRangeException(string, string) {#constructor_6}

```csharp
public AsposeArgumentOutOfRangeException(string message, string paramName)
```

| Параметр | Тип | Описание |
| --- | --- | --- |
| сообщение | String |  |
| paramName | String |  |

### Смотрите также

* class [AsposeArgumentOutOfRangeException](../../asposeargumentoutofrangeexception)
* пространство имен [Aspose.Email](../../asposeargumentoutofrangeexception)
* сборка [Aspose.Email](../../../)

---

## AsposeArgumentOutOfRangeException(Exception) {#constructor_1}

Инициализирует новый экземпляр класса[`AsposeArgumentOutOfRangeException`](../../asposeargumentoutofrangeexception) со ссылкой на внутренний исключение, которое является причиной этого исключения.

```csharp
public AsposeArgumentOutOfRangeException(Exception innerException)
```

| Параметр | Тип | Описание |
| --- | --- | --- |
| innerException | Exception | Исключение, являющееся причиной текущего исключения, или пустая ссылка (ничего в Visual Basic), если внутреннее исключение не указано. |

### Смотрите также

* class [AsposeArgumentOutOfRangeException](../../asposeargumentoutofrangeexception)
* пространство имен [Aspose.Email](../../asposeargumentoutofrangeexception)
* сборка [Aspose.Email](../../../)

<!-- DO NOT EDIT: generated by xmldocmd for Aspose.Email.dll -->
